testing/src/main/java/io/grpc/internal/TestingAccessor.java

```diff
@@ -16,26 +16,38 @@
 
 package io.grpc.internal;
 
-import com.google.instrumentation.stats.StatsContextFactory;
+import io.opencensus.stats.StatsRecorder;
+import io.opencensus.tags.Tagger;
+import io.opencensus.tags.propagation.TagContextBinarySerializer;
 
 /**
  * Test helper that allows accessing package-private stuff.
  */
 public final class TestingAccessor {
   /**
-   * Sets a custom {@link StatsContextFactory} for tests.
+   * Sets a custom stats implementation for tests.
    */
-  public static void setStatsContextFactory(
-      AbstractManagedChannelImplBuilder<?> builder, StatsContextFactory factory) {
-    builder.statsContextFactory(factory);
+  public static void setStatsImplementation(
```

Comment:
These two methods should take the form of the corresponding methods in the builders.

Reply:
Done